frontend/src/components/browse-sightings-page/SightingCard.tsx

```diff
@@ -11,8 +11,19 @@ export const SightingCard: React.FunctionComponent<SightingProps> = ({
 }) => {
   return (
     <div className="sighting-card">
-      <h3 className="fade-in">Sighting #{sighting.id}</h3>
+      {sighting.species?.name ? (
+        <h3 className="sighting-card__title fade-in">
+          Sighting of {sighting.species?.name}
+        </h3>
+      ) : (
+        <h3 className="sighting-card__title fade-in">
+          Sighting recorded by {sighting.seenBy}
+        </h3>
+      )}
 
+      <p className="sighting-card__date fade-in">
+        {new Date(sighting.seenOn).toDateString()}
```

Comment:
Prefer use of [`date-fns`](https://date-fns.org/), [`format`](https://date-fns.org/v2.29.3/docs/format), and possibly [`parse`](https://date-fns.org/v2.29.3/docs/parse) (which should be already installed in `main`, so please merge from there before doing anything else!)

Also in `main` is a change that, I _think_, changes the type of `sighting.seenOn` from a `string` to a `Date()`, so you don't need to parse it from a string first!

Reply:
done now, thanks